README.md

```diff
@@ -1,11 +1,26 @@
 # Bitcoin Armory Website
+## Overview
+This is the website for the community-developed version of [Bitcoin Armory](https://btcarmory.com/), with lead development by [goatpig](https://github.com/goatpig/BitcoinArmory). This site uses the following technologies.
 
-This is the website for the Open Source version of Bitcoin Armory being developed by goatpig. This site uses the [jekyll-now theme](https://github.com/barryclark/jekyll-now)
+- [Ruby](http://www.ruby-lang.org/en/)
+- [Jekyll](https://jekyllrb.com/)
+- [Bundler](https://bundler.io/)
+- [github-pages](https://pages.github.com/)
+- [The *jekyll-now* theme](https://github.com/barryclark/jekyll-now)
+
+While goatpig, the lead Armory maintainer, controls the web site, users are encouraged to contribute updates to the web site (e.g., tips for novice users). goatpig maintains strict control of the Armory repo but is more of a "sanity checker" for the web site, simply making sure the [pull requests](https://github.com/goatpig/BitcoinArmory/tree/gh-pages) are sensible.
```

Comment:
Why is "pull requests" linked to the branch?

Reply:
I just wanted a place to list the actual repo. I could probably reword things a bit and remove the link from that particular spot.